<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TweaksSettingsMiscView</class>
 <widget class="QWidget" name="TweaksSettingsMiscView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MISCELLANEOUS</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">font: 20px &quot;Avenir&quot;;</string>
     </property>
     <property name="text">
      <string>Miscellaneous</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TouchLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">font: 20px &quot;Avenir&quot;; 
padding-top: 10px; 
padding-bottom: 10;</string>
     </property>
     <property name="text">
      <string>Remove Advertisements</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="1" column="1">
      <widget class="TouchCheckBox" name="hideShop">
       <property name="styleSheet">
        <string notr="true">QCheckBox {
    spacing: 15px;
    font: bold 20px &quot;Avenir&quot;;
}

QCheckBox::indicator:unchecked {
    image: url(:/images/widgets/checkbox_off.png);
}
QCheckBox::indicator:checked {
    image: url(:/images/widgets/checkbox_on.png);
}
QCheckBox::indicator:disabled {
    image: url(:/images/widgets/checkbox_disabled.png);
}
</string>
       </property>
       <property name="text">
        <string>Hide Kobo Shop</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="TouchCheckBox" name="hideWishlist">
       <property name="styleSheet">
        <string notr="true">QCheckBox {
    spacing: 15px;
    font: bold 20px &quot;Avenir&quot;;
}

QCheckBox::indicator:unchecked {
    image: url(:/images/widgets/checkbox_off.png);
}
QCheckBox::indicator:checked {
    image: url(:/images/widgets/checkbox_on.png);
}
QCheckBox::indicator:disabled {
    image: url(:/images/widgets/checkbox_disabled.png);
}
</string>
       </property>
       <property name="text">
        <string>Hide Wishlist</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20px &quot;Avenir&quot;;</string>
     </property>
     <property name="text">
      <string>Uninstall</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TouchLabel" name="label_7">
     <property name="styleSheet">
      <string notr="true">font: 20px &quot;Avenir&quot;; 
padding-top: 10px; 
padding-bottom: 10;</string>
     </property>
     <property name="text">
      <string>Uninstall the Tweaks plugin and reboot the device.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="uninstallButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string>font: 22px Avenir; border: 2px solid black; border-radius: 5px;</string>
     </property>
     <property name="text">
      <string>Uninstall Plugin</string>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>417</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>label_2</zorder>
  <zorder>frame</zorder>
  <zorder>label_3</zorder>
  <zorder>uninstallButton</zorder>
  <zorder>frame_3</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TouchLabel</class>
   <extends>QLabel</extends>
   <header>TouchLabel.h</header>
  </customwidget>
  <customwidget>
   <class>TouchCheckBox</class>
   <extends>QCheckBox</extends>
   <header>TouchCheckBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
